test/cypress/plugins/stubs.js

```diff
@@ -498,5 +498,27 @@ module.exports = {
         }]
       }
     ]
+  },
+  postGovUkPayAgreement: (opts) => {
+    return [
+      {
+        predicates: [{
+          equals: {
+            method: 'POST',
+            path: `v1/api/services/${opts.external_id}/govuk-pay-agreement`,
+            headers: {
+              'Accept': 'application/json'
+            },
+            body: { user_external_id: opts.user_external_id }
```

Comment:
Even though its really simple, this should use a fixture which is also used by the pact test.